### PR TITLE
flying numbers in memory

### PR DESCRIPTION
**Pull request by privat**
when playing, the number of the shape is shown.

memory02

### CODE REVIEW

File: contrib/memory/src/memory.nit [hunk 1]
```diff
@@ -62,7 +62,7 @@ class Button
 	# height the display
 	var h: Float = 0.0
 
-	# Event time to live (from 1.0 downto 0.0)
+	rr# Event time to live (from 1.0 downto 0.0)
```
Comment (xymus) [inline contrib/memory/src/memory.nit]: ?

Comment (xymus) [inline contrib/memory/src/memory.nit]: This may be reported as an unknown annotation...

Comment (privat) [inline contrib/memory/src/memory.nit]: uh?

**Comment by xymus** [comment]
+1

Why this change? Is it easier with numbers or is it to teach numbers?

**Comment by privat** [comment]
Both, but the original idea was to make the game more easy to grasp.
Especially when the schema is shown by the NPC cursor.

Le sam. 6 févr. 2016 18:24, Alexis Laferrière <notifications@github.com> a
écrit :

**Pull request merged**